### PR TITLE
Fix/tao 8623 dynamic component

### PR DESCRIPTION
**Pull request by anshul119**
Ticket: https://oat-sa.atlassian.net/browse/TAO-8623

I have updated the version of tao core UI

Related PR:

https://github.com/oat-sa/tao-core-ui-fe/pull/35

How to test:

- Checkout the branch and install node modules in views folder (it will install a custom branch for tao-core-ui package)
- Also checkout the correct branch in taoQtiTest extension (https://github.com/oat-sa/extension-tao-testqti/pull/1541/files)
- Login as a test taker on any instance
- After starting the test, try to use Magnifier, Calculator, Mask or any other test runner tool.
- The UI of the test runner should not be broken anymore as shown in the ticket

What to do after the test?

- [ ] Merge this PR: https://github.com/oat-sa/tao-core-ui-fe/pull/35
- [ ] Change tao-coreui the version from branch to correct version in package (0.18.0)
- [ ] Install packages and commit package lock and package

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "@babel/polyfill": "^7.4.4",
     "@oat-sa/tao-core-libs": "~0.1.0",
     "@oat-sa/tao-core-sdk": "~0.5.0",
-    "@oat-sa/tao-core-ui": "~0.17.0",
+    "@oat-sa/tao-core-ui": "oat-sa/tao-core-ui#fix/tao-8623-dynamic-component",
```
Comment (AntonTsymuk) [inline views/package.json]: I think repository should be `oat-sa/tao-core-ui-fe`

**Comment by AntonTsymuk** [comment]
on my local env the area masking tool is still corrupted
Screen Shot 2019-07-02 at 7 01 04 PM

**Comment by AntonTsymuk** [comment]
did you check formulas and help tools?

File: views/package.json (hunk 1 above)
Comment (btamas) [reply views/package.json]: It should be the version after https://github.com/oat-sa/tao-core-ui-fe/pull/35 will be released.

**Review by btamas — APPROVED** [review]

File: views/package.json (hunk 1 above)
Comment (anshul119) [reply views/package.json]: Correct, my bad

**Comment by anshul119** [comment]
> on my local env the area masking tool is still corrupted
> Screen Shot 2019-07-02 at 7 01 04 PM

Have you checked out the right branches in tao-core, taoQtiTest and correct package installed in tao/views ?

**Review by atsymuk — APPROVED** [review]

**Pull request merged**